code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -301,11 +301,6 @@ There are several things that need to be remembered:
 		if(!t_state)
 			t_state = s_store.icon_state
 		overlays_standing[SUIT_STORE_LAYER]	= mutable_appearance('icons/mob/belt_mirror.dmi', t_state, -SUIT_STORE_LAYER)
-		var/mutable_appearance/s_store_overlay = overlays_standing[SUIT_LAYER]

```

Comment:
Another issue is right here, it was applying whole suit sprite instead of just belt sprite.

Reply:
Yeah, that's why I removed it. Also, wasn't the subsequent code relying upon the variable that was just defined? This would make line 305 through 308 useless because all it does is adjust the suit layer. Additionally, it wouldn't affect the belt layer because it's inside the suit.

Reply:
if you want to remove the feature, open proper removal PR because this only cuts a part of it without cleaning it up, if you want to fix the typo this this is not a it.